README.md

```diff
@@ -237,10 +238,14 @@ Converts the passed object which contains primitive values into the query string
 
 ### [omit](./lib/omit.ts)
 
-Returns the passed object without the passed keys.
+Gets the passed object and returns the new one but without the passed keys.
 
 E.g. in React: `omit(this.props, 'mods', 'mix')`.
 
+### [pick](./lib/pick.ts)
```

Comment:
Even though the reversed helper is called “omit”, I guess “pick” is a better name than “permit”.